.github/workflows/build-liimstrap.yml

```diff
@@ -0,0 +1,31 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+  workflow_dispatch:
+
+jobs:
+
+  release:
+
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Build liimstrap image
+      run: docker build -t ustclug/liimstrap:liims-2 .
+        
+    - name: Pack to squashfs file
+      run: docker run --privileged --cap-add=SYS_ADMIN --rm -v /srv/dest:/srv/dest -e ROOT_PASSWORD=${{ secrets.LIIMSTRAP_ROOT_PASSWORD }} -e SQUASHFS=true ustclug/liimstrap:liims-2
+    
+    - name: Set release tag env
+      run: |
+        echo "TAG_NAME=$(date +'%Y-%m-%d')_$(echo $GITHUB_SHA | cut -c 1-6)" >> $GITHUB_ENV
+        
+    - uses: ncipollo/release-action@v1
+      with:
+        artifacts: "/srv/dest/lims$(date+'%y%m%d')/vmlinuz,/srv/dest/lims$(date+'%y%m%d')/initrd.img,/srv/dest/lims$(date+'%y%m%d')/root.sfs"
```

Comment:
Does shell command expansion really work here? I'd rather edit `deploy` script and remove the extra directory altogether.

Reply:
Not working. Will changing dest directory break your things? If not, I will try to edit the script.